12-offer-encoding.md

```diff
@@ -419,15 +410,16 @@ The writer:
         - MUST specify `invreq_amount`.`msat` as greater or equal to amount expected by `offer_amount` (and, if present, `offer_currency` and `invreq_quantity`).
     - MUST set `invreq_payer_id` to a transient public key.
     - MUST remember the secret key corresponding to `invreq_payer_id`.
-    - if `offer_quantity_min` or `offer_quantity_max` are present:
+    - if `offer_quantity_max` is present:
       - MUST set `invreq_quantity`
-      - MUST set it within that (inclusive) range.
+      - if `offer_quantity_max` is non-zero:
+        - MUST set `invreq_quantity` less than or equal to `offer_quantity_max`.
```

Comment:
Since there is no longer an `offer_quantity_min`, we should also say that `invreq_quantity` MUST be greater than 0.